app/lib/actions/pulp3/orphan_cleanup/delete_orphan_remotes.rb

```diff
@@ -0,0 +1,16 @@
+module Actions
+  module Pulp3
+    module OrphanCleanup
+      class DeleteOrphanRemotes < Pulp3::AbstractAsyncTask
+        def plan(smart_proxy)
+          plan_self(:smart_proxy_id => smart_proxy.id)
+        end
+
+        def invoke_external_task
+          smart_proxy_service = ::Katello::Pulp3::SmartProxyRepository.new(smart_proxy)
+          smart_proxy_service.delete_orphaned_remotes_for_mirror_proxies(smart_proxy)
```

Comment:
i don't think this method takes a parameter:

```
ArgumentError

wrong number of arguments (given 1, expected 0)

---
- "/home/vagrant/git/katello/app/services/katello/pulp3/smart_proxy_repository.rb:41:in
  `delete_orphaned_remotes_for_mirror_proxies'"
```

Reply:
Yeah, that's the same error as the previous one, but mirrored on the other action.